srcpkgs/python3-CairoSVG/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'python3-CairoSVG'
+pkgname=python3-CairoSVG
+version=2.5.2
+revision=1
+wrksrc="${pkgname#*-}-${version}"
+build_style=python3-module
+hostmakedepends="python3-setuptools python3-pip"
```

Comment:
remove pip and try to build and run tests, you will see then

Reply:
I saw this error: `/usr/bin/python3: No module named pip` and this is why I added python3-pip...

Reply:
Because it fails to import a dependency, so it tries to download it with pip and fails, we need to include the dependency instead of using pip and producing a package with broken dependencies.

Reply:
Ah okay, thank you! I will try to fix it